src/features/common/ui/Navbar.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useState,useEffect} from "react";
+
+export default function Navbar() {
+
+    const [isMobile, setIsMobile] = useState<boolean>(window.innerWidth < 768);
+    const [isMobileNavOpen, setIsMobileNavOpen] = useState<boolean>(false);
+    const handleClick = () => setIsMobileNavOpen(!isMobileNavOpen);
+
+    useEffect(() => {
```

Comment:
Shall we use useLayoutEffect ?

https://kentcdodds.com/blog/useeffect-vs-uselayouteffect